src/components/KSelect/KSelectItem.vue

```diff
@@ -99,60 +104,62 @@ const handleClick = (e: MouseEvent): void => {
     }
 
     .k-select-item-label {
-      color: var(--grey-600);
-      font-size: 14px;
-      font-weight: 500;
-      line-height: 20px;
-      padding: 8px;
+      color: var(--grey-600, var(--kui-color-text-primary-strong, $kui-color-text-primary-strong));
+      font-size: var(--kui-font-size-30, $kui-font-size-30);
+      font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
```

Comment:
issue: regular font weight is `400` -- you need `500` here

Reply:
Fixed